src/main/resources/com/google/jenkins/plugins/computeengine/InstanceConfiguration/help-windowsPrivateKeyCredentialsId.html

```diff
@@ -0,0 +1,30 @@
+<div>
+    <p>
+    Download the SSH Credentials Plugin. Then create an SSH Username with private key credential with Global scope.<br/>
+
```

Comment:
Add <br/> if you want this to be formatted as a line break, otherwise remove the '\n'

Reply:
done